packages/flutter_test/test/widget_tester_test.dart

```diff
@@ -662,6 +663,41 @@ void main() {
       expect(find.text('Item 15', skipOffstage: true), findsOneWidget);
     });
   });
+
+  testWidgets('verifyTickersWereDisposed control test', (WidgetTester tester) async {
+      FlutterError error;
```

Comment:
nit: indents are off here

Reply:
yes sorry for that :( should I make new PR to fix this?

Reply:
no worries. go ahead and create a new PR with just the whitespace fix. Good news is whitespace only CLs are really quick to review. :)